core/src/main/java/org/infinispan/xsite/BackupSenderImpl.java

```diff
@@ -244,21 +173,18 @@ private void sendTo(XSiteReplicateCommand command, Collection<XSiteBackup> xSite
       if (rpcManager == null) {
          for (XSiteBackup backup : xSiteBackups) {
             XSiteResponse cs = transport.backupRemotely(backup, command);
+            takeOfflineManager.registerRequest(cs);
```

Comment:
Why do we no need to register the request when we didn't before? AFAICT, there was no call to `XSiteResponse#whenCompleted` previously

Reply:
the previous one is here
https://github.com/infinispan/infinispan/blob/master/core/src/main/java/org/infinispan/xsite/BackupSenderImpl.java#L418

The old method updates the offline status and handles the exception. in this PR, it only handles the exception.